ext/src/dll/dllmain.cc

```diff
@@ -6,9 +6,8 @@
 // Include API header files here for exporting
 #include <opentelemetry/trace/provider.h>
 
-#if defined(ENABLE_LOGS_PREVIEW)
-#  include <opentelemetry/logs/logger_provider.h>
-#endif  // ENABLE_LOGS_PREVIEW
+#include <opentelemetry/logs/logger_provider.h>
+#include <opentelemetry/logs/provider.h>
```

Comment:
nit - do we need this header ?

Reply:
Yes, it is needed. If the provider header file is not included in the main source, then it is only available in some library which will not be put into the final DLL as it will be considered as dead code. Reference it in the main source will cause it to be added to the final image.